app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:id])
+    @comment = current_user.comments.new(text: params[:comment])
+    @comment.post_id = post.id
+    @comment.save
+    redirect_to "/user/#{current_user.id}/post/#{post.id}"
+  end

```

Comment:
- [ ] when i submit a comment, it does not get added to the post, this is also same as when I like a post, kindly fix it
![Screenshot from 2022-08-26 15-38-34](https://user-images.githubusercontent.com/66097371/186930028-05172a2f-5293-4ae7-a431-aebd300efbb1.png)